crates/cli/tests/benchmarks/platform/PlatformTasks.roc

```diff
@@ -0,0 +1,9 @@
+hosted PlatformTasks
+    exposes [putLine, putInt, getInt]
+    imports []
+
+putLine : Str -> Task {} *
+
+putInt : I64 -> Task {} *
+
+getInt : Task { value : I64, isError : Bool } *
```

Comment:
Should these be changed? [context](https://roc.zulipchat.com/#narrow/stream/316715-contributing/topic/Task.20as.20a.20built-in/near/465046819)

Reply:
Or should we just take care of that when we add the error generation for these return types?

Reply:
Used {} instead of *